associations/urls.py

```diff
@@ -36,6 +36,7 @@
 router.register(r'transactions', TransactionViewSet)
 router.register(r'funding', FundingViewSet)
 urlpatterns += [
+    path("associations/<association_pk>/image", set_association_image),
```

Comment:
Is the omission of the trailing `/` intentional?

Reply:
nope